.prettierignore

```diff
@@ -1,2 +1,3 @@
 src/ui/public
 src/ui/generated_reports
+src/api/helpers/cron/scheduler.ts
```

Comment:
Do we need this - I can have a look at the linting issue, try to fix it

Reply:
Sure, if it can be fixed I can remove this

Reply:
Our eslint config is slightly different to DTFS.

I got this the `cron/scheduler.ts` file passing with prettier by updating our `eslintrc.js` - in the `extends` array;

Replacing these:

```js
'plugin:prettier/recommended',
'prettier/prettier',
```

With this (like DTFS):

```js
pretttier
```

...And also removing the `prettier/prettier` config from `rules`.

I'm not sure about any other consequences of this though, need to investigate.